src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -307,12 +307,6 @@ export function ChangeDataView({
           isTextBasedLangSelected={isTextBasedLangSelected}
           setPopoverIsOpen={setPopoverIsOpen}
           onChangeDataView={async (newId) => {
-            try {
-              // refreshing the field list
-              await dataViews.get(newId, false, true);
-            } catch (e) {
-              //
-            }
```

Comment:
With this PR this is no longer needed. it was introduced to enable new fields of data views showing up in the field list, that were added after the last cached version. With this PR this is redundant, resolving #169360 

The benefit is, user no longer need to wait until the field list is refreshed when data views are switched. There was also no loading indicator being displayed . Now the unified field list is showing new fields, and when they are loaded, a loading indicator is being displayed. Better UX it should be